### PR TITLE
Update tests for RHODS Alerts

### PR DESCRIPTION
**Pull request by VaishnaviHire**
Jira Issue: https://issues.redhat.com/browse/RHODS-4071

### CODE REVIEW

**Comment by VaishnaviHire** [comment]
cc: @jgarciao

File: tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot [hunk 1]
```diff
@@ -99,125 +103,92 @@ Verify Alert RHODS-PVC-Usage-At-100 Is Fired When User PVC Is At 100 Percent
...
 
-Verify Alert "RHODS Route Error Burn Rate" Is Fired When Traefik Is Down    # robocop: disable:too-long-test-case
-    [Documentation]    Verifies that alert "RHODS Route Error Burn Rate" is fired when traefik-proxy is not working
+Verify Alerts Are Fired When Traefik Is Down    # robocop: disable:too-long-test-case
+    [Documentation]    Verifies that alerts "RHODS JupyterHub Probe Success Burn Rate" and "RHODS Probe
+    Success Burn Rate" alerts are fired when traefik-proxy is not working
     [Tags]    Tier3
     ...       ODS-738
 
     Skip    msg=This alert was disabled in RHODS 1.3.0. More info at RHODS-2101
```
Comment (jgarciao) [inline tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot:112]: This line should be removed or the test will be skipped always

**Review by jgarciao — CHANGES_REQUESTED** [review]

File: tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot [hunk 2]
```diff
@@ -99,125 +103,92 @@ Verify Alert RHODS-PVC-Usage-At-100 Is Fired When User PVC Is At 100 Percent
 
     [Teardown]    Teardown PVC Alert Test
 
-Verify Alert "RHODS Route Error Burn Rate" Is Fired When Traefik Is Down    # robocop: disable:too-long-test-case
-    [Documentation]    Verifies that alert "RHODS Route Error Burn Rate" is fired when traefik-proxy is not working
+Verify Alerts Are Fired When Traefik Is Down    # robocop: disable:too-long-test-case
+    [Documentation]    Verifies that alerts "RHODS JupyterHub Probe Success Burn Rate" and "RHODS Probe
+    Success Burn Rate" alerts are fired when traefik-proxy is not working
```
Comment (jgarciao) [inline tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot:108]: It should be (3 dots followed by 4 spaces) `...    Success Burn Rate" alerts are fired when traefik-proxy is not working`

File: tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot [hunk 3]
```diff
@@ -99,125 +103,92 @@ Verify Alert RHODS-PVC-Usage-At-100 Is Fired When User PVC Is At 100 Percent
...
     ...    alert-duration=120
     ...    timeout=5 min
 
     [Teardown]    ODS.Restore Default Deployment Sizes
 
-Verify Alert "RHODS Route Error Burn Rate" Is Fired When RHODS Dashboard Is Down    # robocop: disable:too-long-test-case
-    [Documentation]    Verifies that alert "RHODS Route Error Burn Rate" is fired when rhods-dashboard is not working
+Verify Alerts are Fired When RHODS Dashboard Is Down    # robocop: disable:too-long-test-case
+    [Documentation]    Verifies that alert "RHODS Dashboard Route Error Burn Rate" and "RHODS Probe Success Burn Rate"
+    are fired when rhods-dashboard is not working
```
Comment (jgarciao) [inline tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot:157]: It should be (3 dots followed by 4 spaces) `... are fired when rhods-dashboard is not working`

**Review by jgarciao — CHANGES_REQUESTED** [review]

File: tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot (hunk 2 above)
Comment (VaishnaviHire) [reply tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot]: Updated

File: tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot (hunk 3 above)
Comment (VaishnaviHire) [reply tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot]: Updated

**Comment by jgarciao** [comment]
Being tested here: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1054/

**Comment by jgarciao** [comment]
Being tested here: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1056/

File: tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot [hunk 4]
```diff
@@ -25,45 +25,50 @@ ${TEST_ALERT_PVC100_NOTEBOOK_PATH}      SEPARATOR=
...
 
 Verify No Alerts Are Firing Except For DeadManSnitch    # robocop: disable:too-long-test-case
     [Documentation]    Verifies that, in a regular situation, only the DeadManSnitch alert is firing
     [Tags]    Smoke
     ...       Tier1
     ...       ODS-540
-
-    Verify Alert Is Firing And Continue On Failure
-    ...    DeadManSnitch    DeadManSnitch
+    ...       RunThisTest
```
Comment (jgarciao) [inline tests/Tests/200__monitor_and_manage/210__alerts/210__alerts.robot:51]: `RunThisTest` tag should be removed

**Review by jgarciao — CHANGES_REQUESTED** [review]

**Review by bdattoma — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Pull request merged**